app/components/service/serviceController.js

```diff
@@ -35,79 +36,168 @@ function ($scope, $stateParams, $state, Service, ServiceHelper, Task, Node, Mess
     service.EditReplicas = false;
   };
 
+  $scope.goToItem = function(hash) {
+    $location.hash(hash);
```

Comment:
It's possible to use `$anchorScroll` without the call to `$location.hash(hash)`. Just use `$anchorScroll(hash)`, this will not update the URL with the hash, but is that really useful ?